src/pages/SingleCell/PieToolTip.tsx

```diff
@@ -0,0 +1,393 @@
+import React, { useState, useEffect, useRef } from 'react';
+import { VictoryPie, VictoryTooltip } from 'victory';
+import { handleDownloadSVG, handleDownloadPDF } from './downloadUtils';
+import './styles.css';
+import { observer } from 'mobx-react-lite';
+
+// Define the DataBin interface
+
+// Define props interface for the Chart component
+interface PatientData {

```

Comment:
I think you are defining `interface PatientData` three times in you code. You can define it once as `export interface PatientData` and import it when needed.